2019-04-26 02:28:58,350 ERROR: Exception on /camera [GET] [in /home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/app/routes.py", line 47, in camera
    return render_template("camera.html", title=camera)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/app/templates/camera.html", line 1, in top-level template code
    {% extends "base_boots" %}
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/home/ahmnouira/Desktop/blog_flask/Ahmed_blog/ahmed-nouira-blog4-V0.1.0/venv/lib/python3.5/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: base_boots
